erpnext_ocr/erpnext_ocr/doctype/ocr_read/ocr_read.py

```
@@ -28,6 +31,9 @@ def read_document(path, lang='eng'):
     if path is None:
         return None
 
+    if not lang_is_supported(lang):
+        return "The selected language is not available. Please contact your administrator."
```

@AminovE99 That's a pretty bad practice to return a hard coded string if language is not supported.
We should throw an exception in this case, with a translated error message (ie. call the translation function `_`).